packages/beacon-node/src/chain/factory/block/body.ts

```diff
@@ -175,28 +162,45 @@ export async function prepareExecutionPayload(
   finalizedBlockHash: RootHex,
   state: CachedBeaconStateBellatrix,
   suggestedFeeRecipient: string
-): Promise<PayloadId | null> {
```

Comment:
`prepareExecutionPayload` is used over here as well: https://github.com/ChainSafe/lodestar/blob/unstable/packages/beacon-node/src/chain/prepareNextSlot.ts#L116 as  a way to early signal the execution layer to produce the blocks so that transactions can be bundled, this change will break that